.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug about the Scala compiler or the standard library
+title: ''
+assignees: ''
+
+---
+
+## minimized code
+
+```Scala
+println("hello, world")
```

Comment:
super tiny nitpick: s/Scala/scala/, to match the one below. unless you'd rather make them match the other way